scripts/generate_strings.py

```diff
@@ -165,6 +165,10 @@ def serialize(string):
                     output.write(f"    //: {comment}\n")
                 output.write(f"    QT_TRID_NOOP(\"{string['stringId']}\"),\n\n")
 
+            # This is done to make windows compiler happy
+            if len(stringIds) == 0:
+                output.write(f"    QT_TRID_NOOP(\"vpn.dummy.ignore\"),\n\n")
```

Comment:
This is really not great, as we'll end up with this placeholder exposed for localization in the pipeline.

If there's no other way around (can we avoid generating these files if there are no strings?), I would suggest to move an actual string to `strings.yaml` instead.